packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -1,6 +1,8 @@
 import type { EventTypeCustomInput, EventType, Prisma, Workflow } from "@prisma/client";
 import type { z } from "zod";
 
+import { fieldsThatSupportLabelAsSafeHtml } from "@calcom/features/form-builder/fieldsThatSupportLabelAsSafeHtml";
+import { markdownToSafeHTML } from "@calcom/lib/markdownToSafeHTML";
```

Comment:
It is okay to include markdownToSafeHtml here because this module isn't imported at client side.